tests/cucumber/support/widgets/threejs_editor_widget.js

```diff
@@ -7,36 +7,100 @@ import {SELECTORS} from "../selectors";
 export class ThreeJSEditorWidget extends Widget {
     constructor(selector) {
         super(selector);
-        this._selectros = SELECTORS.threeJSEditorWidget;
+        this._selectors = SELECTORS.threeJSEditorWidget;
     }
 
     clickOnMenuItem(menuTitle, menuItemTitle) {
-        exabrowser.scrollAndClick(this._selectros.menuByTitle(menuTitle));
-        exabrowser.scrollAndClick(this._selectros.menuItemByTitle(menuItemTitle));
+        exabrowser.scrollAndClick(this._selectors.menuByTitle(menuTitle));
+        exabrowser.scrollAndClick(this._selectors.menuItemByTitle(menuItemTitle));
+    }
+
+    clickOnToolbarButton(title) {
+        const toolbarBtnSelector = this._selectors.toolbarBtnByTitle(title);
+        exabrowser.scrollAndClick(toolbarBtnSelector);
+    }
+
+    makeMultipleSelection({ x1, y1, x2, y2 }) {
```

Comment:
It would be good to add an explanation re what the parameters are - e.g. position with respect to what reference point specifically